frontends/web/src/common/ApiService.js

```diff
@@ -418,8 +418,12 @@ export default class ApiService {
     return this.fetch(export_link, {
       method: "GET",
     }).then((res) => {
-      res = new TextEncoder("utf-8").encode(JSON.stringify(res));
-      return download(res, "export.json", "text/json");
+      if (res.type == "download") {
+        const res_data = new TextEncoder("utf-8").encode(
+          JSON.stringify(res.data)
+        );
+        return download(res_data, "export.json", "text/json");
+      }
```

Comment:
I want to give some feedback to the user i.e a hover popup over the download button that disappears after a few seconds (this will tell the user whether the upload/sending email was successful)

Reply:
Sounds great. You could do this in the "Rounds" section of the task owner interface, where the download button is.